src/components/LandingPage/HackathonInformationContainer.tsx

```diff
@@ -0,0 +1,170 @@
+import Image from "next/image";
+
+function SectionContainer({
+  title,
+  sectionImage,
+  leftPosition,
+  blurb,
+  bgColor,
+  fontColor,
+}: {
+  title: React.ReactNode;
+  sectionImage: string;
+  leftPosition?: boolean;
+  blurb: string;
+  bgColor?: string;
+  fontColor?: string;
+}) {
+  return (
+    <div
+      className={`flex w-full flex-row items-center justify-between gap-16 ${bgColor} ${fontColor} px-[5%] py-20`}
+    >
+      <div className={`flex flex-col gap-4 ${fontColor}`}>
+        {title}
+        <p className=" max-w-lg font-medium">{blurb}</p>
+      </div>
+      <div
+        className={`relative ${leftPosition && "order-first"} box-content hidden aspect-square w-2/3 min-w-48 max-w-96 rounded-lg md:block`}
```

Comment:
Can you use the tailwind-merge library for this? Syntax would be like: `twMerge(....., leftPosition && "order-first", ...)`

Reply:
The twMerge library is giving me a lot of eslint errors and I can't seem to figure out how to fix it. Might need some help on this one